catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/UpdateOperations.java

```diff
@@ -638,7 +638,7 @@ private QueryRequestImpl createQueryRequest(UpdateRequest updateRequest) {
         new QueryImpl(
             queryOperations.getFilterWithAdditionalFilters(idFilters, updateRequest),
             1, /* start index */
-            0, /* page size */
+            QueryOperations.MAX_PAGE_SIZE, /* page size */
```

Comment:
Will the case where we are updating more items than the MAX_PAGE_SIZE fail? We might consider using the ResultIterable to ensure we get all the results

Reply:
Right after I put this up I started wondering the same thing. I will update to use the ResultsIterable

Reply:
After looking at it a little more I think instead of using the MAX_PAGE_SIZE I will just use the size of the request for the page size. It doesn't look like we use the ResultIterable within the standard framework so I think this change is consistent with how we handle queries within the framework right now.